source/_docs/platform-considerations.md

```diff
@@ -164,6 +164,17 @@ For sites that need to provide services with Cross-Origin Resource Sharing (COR
 
 Sites that consume services using CORS, such as Amazon S3 CORS, do work on Pantheon.
 
+For WordPress users, they can use the [WP-CORS plugin](https://wordpress.org/plugins/wp-cors/){.external} or add this in their theme's `function.php`:
+
+```
+add_filter('allowed_http_origins', 'pantheon_allowed_origins');
+
+function pantheon_allowed_origins($urls) {
+    $urls[] = 'https://www.example.com';
```

Comment:
@carl-alberto what would `example.com` be replaced with in this example?

Reply:
@alexfornuto These will be the urls allowed for cross-domain requests, please note that protocol(http/https) and www/non www urls are relevant here. Multiple values is also possible, eg.

`$urls[] = array( 'https://www.example.com', 'http://www.example.com', 'https://example.com', 'http://example.com'  ) ;`